framework/helpers.php

```diff
@@ -56,8 +69,18 @@ function cpt_next_posts_link( WP_Query $wp_query, $label ) {
 	if ( $nextpage <= $max_page ) {
 		$attr = apply_filters( 'next_posts_link_attributes', '' );
 		$attr = apply_filters( 'jtf_next_posts_link_attributes', $attr );
+		if ( ! empty( $load_more_attr ) ) {
+			if ( ! empty( $attr ) && strpos( $attr, 'class=' ) ) {
+				$attr = preg_replace( '/class="/i', 'class="jtf-load-more ', $attr );
+			} elseif ( ! empty( $attr ) ) {
+				$attr .= ' class="jtf-load-more"';
+			} else {
+				$attr = 'class="jtf-load-more"';
+			}
```

Comment:
2 "else" is too much for this case. 1 extra space is not a problem. 
all we need to check that $attr has class or not (btw strpos condition is wrong. if class will be the first one - your code won't work)
if attr has no class - just add. so the code should be much more simple